### PR TITLE
Digest/nushackers redesign

### PR DESCRIPTION
**Pull request by li-kai**
Includes fix for /fh

### CODE REVIEW

**Comment by li-kai** [comment]
Deploy preview ready!

Built with commit c5dc879d2f947c2c2c35d789d5ab9c217013d134

https://deploy-preview-93--nushackers.netlify.com

File: content/post/2017-09-17-digest-nushackers-site-redesign.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+author: Li Kai
+categories:
+- Digest
+date: 2017-09-17T00:00:00Z
+published: true
+title: 'Digest: NUSHackers site redesign'
+url: /2017/09/17/digest-nushackers-site-redesign/
+---
+
+This summer, the NUSHackers site went through a huge redesign (you’re looking at it right now). It took the efforts of the everyone at the core team to make the blog better. In terms of improvements we managed to improve not only the developer experience but also the speed and performance of the website itself. In this week's digest, we will be talking about some of the improvements we've made and how we've done it.
```
Comment (advaypal) [inline content/post/2017-09-17-digest-nushackers-site-redesign.md]: Replace all 'NUSHackers' with 'NUS Hackers'.
Everyone 'on' the core team

File: content/post/2017-09-17-digest-nushackers-site-redesign.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Static site generators
+
+Before we dive into the details, let’s take the time to explain what a static site generator is. A static site generator, as the name implies, generates a website. It is static, because once the html files are generated, they will always look and stay the same, which contrast to dynamic sites such as Facebook that present different content every time you visit. These generators takes a bunch of content files, normally written in a format called markdown, along with some template html files and combine them to make a website. Add in some css for styling, and you’ve got a beautiful website. At NUSHackers, we originally used a generator called [Jekyll]. More alternatives can be found at [staticgen](https://staticgen.com).
```
Comment (advaypal) [inline content/post/2017-09-17-digest-nushackers-site-redesign.md]: 'which is in contrast'

File: content/post/2017-09-17-digest-nushackers-site-redesign.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+## Faster site & build previews
+
+The last feature that we added was to move our hosting to a static site called [Netlify](https://netlify.com).
+
+<img src="/img/2017/09/netlify.png" alt="netlify"/>
+
+Not only did it improve the load time of our site through their incredibly fast Content-Distribution Networks (CDN), they also provided build previews whenever we pushed to to github. This meant that any new changes could be previewed in an actual fork of the site. It certainly made reviewing a lot easier and more trustworthy.
+
+When all said and done, the new blog is incredibly, fast, convenient and beautiful. We hope that this digest encouraged you to start your own. Our code is always publically available at [Github](github.com/nushackers/nushackers-site) and you are free to copy if for your own use!
```
Comment (advaypal) [inline content/post/2017-09-17-digest-nushackers-site-redesign.md]: - Remove 'When'
- 'Publicly'
- 'it for your own use'

**Comment by donjar** [comment]
Nitpick: you probably want to capitalize all instances of HTML, CSS, and SCSS :)

File: content/post/2017-09-17-digest-nushackers-site-redesign.md (hunk 3 above)
Comment (donjar) [reply content/post/2017-09-17-digest-nushackers-site-redesign.md]: Also "incredibly, fast" --> "incredibly fast"

**Pull request merged**